frontend/src/__tests__/cypress/cypress/pages/pipelines/createRunPage.ts

```diff
@@ -146,28 +140,12 @@ export class CreateRunPage {
     this.findParamById(id).type(value, { parseSpecialCharSequences: false });
   }
 
-  selectExperimentByName(name: string): void {
-    this.findExperimentSelect()
-      .click()
-      .get('[data-id="experiment-selector-table-list"]')
-      .findByText(name)
-      .click();
-  }
-
-  selectPipelineByName(name: string): void {
-    this.findPipelineSelect()
-      .click()
-      .get('[data-id="pipeline-selector-table-list"]')
-      .findByText(name)
-      .click();
+  selectExperimentByName(name: string): Cypress.Chainable<JQuery<HTMLTableCellElement>> {
+    return cy.findByTestId('experiment-selector-table-list').find('td').contains(name).click();
   }
```

Comment:
Action functions should not return a value as each action should be performed separately.